test/integration/notifications/notification.spec.ts

```diff
@@ -1,8 +1,9 @@
 import { Prices } from "@app"
 import { getRecentlyActiveAccounts } from "@app/accounts/active-accounts"
+import { sendBalanceToAccounts } from "@app/accounts/send-balance-to-accounts"
```

Comment:
is not mandatory but if we want to verify honeycomb with local data then this should be used through wrapped functions